assets/src/application/ui/layout/entityList/filterBar/buttons/ToggleSortingButton.tsx

```diff
@@ -20,16 +20,16 @@ const ToggleSortingButton: React.FC<ToggleSortingButtonProps> = ({
 
 	return (
 		<IconButton
-			className={ className }
-			icon={ () => <Sort color={ sortingEnabled ? '#fff' : null } /> }
-			id={ id }
-			label={ __('sorting') }
-			onClick={ toggleSorting }
-			tooltip={ tooltip }
-			tooltipProps={ { placement: 'top' } }
-			labelClassName={ 'ee-filter-bar__btn-wrap' }
-			labelPosition={ LabelPosition.BOTTOM_CENTER }
-			{ ...rest }
+			className={className}
+			icon={Sort}
+			id={id}
+			label={__('sorting')}
+			onClick={toggleSorting}
+			tooltip={tooltip}
+			tooltipProps={{ placement: 'top' }}
```

Comment:
Lets avoid object literals for props, they cause re-renders for being created everytime.